pkgs/by-name/go/go-passbolt-cli/package.nix

```
@@ -0,0 +1,53 @@
+{ buildGoModule, fetchFromGitHub, installShellFiles, lib }:
+
+buildGoModule rec {
+  pname = "go-passbolt-cli";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "passbolt";
+    repo = "go-passbolt-cli";
+    rev = "v${version}";
+    hash = "sha256-I+niNUowKTFDMa7yOnRToMFPzO/CbnjXHJr5nAhhHcg=";
+  };
+
+  vendorHash = "sha256-XRHGq3Qeq7VWHzw5WWVv4x5orQu740lttGVreiu7qP4=";
+
+  ldflags = [
+    "-X=main.version=${version}"
+    "-X=main.commit=${src.rev}"
+    "-X=main.date=1970-01-01T00:00:00Z"
+  ];
+
+  doCheck = false;
```

```suggestion
  doCheck = false;
```

Please add explanation comment here, such as `# requiring network access`

---

I'm sorry for my ignorance, what is `doCheck` actually doing?
Maybe we can do checks...

---

(And thanks a lot for the review!)

---

It is normally not needed, until you see check failures you will know what to do...

---

Ok, I'll test. Thank you.

---

Went all right...